Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeViewController.swift

```diff
@@ -273,7 +273,7 @@ class CreateSafeViewController: UIViewController, UITableViewDelegate, UITableVi
             self.chain = self.uiModel.chain
             self.initExecutionBuilder()
             if self.chain.isSupported(feature: .relayingMobile) {
-                self.getRemainingRelays()
+                self.getRemainingRelays(enableRelay: true)
```

Comment:
How can "getRemainingRelays" be called if the relay is not enabled? I think it's not the proper fix